tuxemon/core/components/event/actions/random_encounter.py

```diff
@@ -130,3 +99,40 @@ def start(self):
     def update(self):
         if self.game.get_state_name("CombatState") is None:
             self.stop()
+
+
+def _choose_encounter(encounters, total_prob):
+    total = 0
+    roll = random.random() * 100
```

Comment:
The previous implementation was buggy:

1. Random number was 1 - 1000 so the probability was 10% of what it should have been.
2. Even with the correct range, the probabilities were lower than they should have been.